includes/REST_API/Page_Template_Controller.php

```diff
@@ -75,4 +75,31 @@ public function get_collection_params() {
 
 		return $query_params;
 	}
+
+	/**
+	 * Retrieves the attachment's schema, conforming to JSON Schema.
+	 *
+	 * Removes some unneeded fields to improve performance by
+	 * avoiding some expensive database queries.
+	 *
+	 * @since 1.10.0
+	 *
+	 * @return array Item schema as an array.
+	 */
+	public function get_item_schema(): array {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+
+		$schema = parent::get_item_schema();
+
+		unset(
+			$schema['properties']['permalink_template'],
+			$schema['properties']['generated_slug']
+		);
+
+		$this->schema = $schema;
+
+		return $this->add_additional_fields_schema( $this->schema );
+	}
```

Comment:
This should really be moved to `Stories_Base_Controller`

Reply:
No, because we need `permalink_template` in `Stories_Controller` for the editor.

Reply:
Should it also be added to `Page_Template_Controller` then?

Reply:
This is in `Page_Template_Controller` :)